### PR TITLE
[Merged by Bors] - feat(SetTheory/Cardinal/Aleph): simpler definition of `aleph`

### PR DESCRIPTION
**Pull request by vihdzp**
We redefine `aleph' o = (omega' o).card` and `aleph o = (omega o).card`.

---

- [x] depends on: #17669

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Cardinal/Cofinality.lean [hunk 1]
```diff
@@ -660,16 +660,21 @@ theorem aleph0_le_cof {o} : ℵ₀ ≤ cof o ↔ IsLimit o := by
...
+theorem cof_preOmega {o : Ordinal} (ho : o.IsLimit) : (preOmega o).cof = o.cof :=
+  isNormal_preOmega.cof_eq ho
+
+@[simp]
+theorem cof_omega {o : Ordinal} (ho : o.IsLimit) : (ω_ o).cof = o.cof :=
+  isNormal_omega.cof_eq ho
 
 set_option linter.deprecated false in
-@[deprecated preAleph_cof (since := "2024-10-22")]
+@[deprecated cof_preOmega (since := "2024-10-11")]
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Cardinal/Cofinality.lean:671]: Something weird going on here

**Review by YaelDillies — COMMENTED** [review]
Is `preAleph (o + ω) = ℵ_ o` still defeq?

File: Mathlib/SetTheory/Cardinal/Cofinality.lean (hunk 1 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Cardinal/Cofinality.lean]: Should be fixed now.

**Comment by YaelDillies** [comment]
Okay great!

maintainer merge

**Pull request closed without merging**